src/Estonianpin/Utils/Utils.php

```diff
@@ -239,4 +239,37 @@ public function generateRandomDateArray(): array {
         ];
     }
 
+    /**
+     * @param \DateTime $start
+     * @param \DateTime $end
+     * @return Iterable
+     */
+    public function getPinsGeneratorForRange(\DateTime $start, \DateTime $end): iterable
+    {
```

Comment:
The current project uses Java style formatting. Curly braces start  on the same line as function name.